native/packages/react-native-bpk-component-picker/src/BpkPickerItem.android.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import PropTypes from 'prop-types';
+import BpkText from 'react-native-bpk-component-text';
+import BpkTouchableNativeFeedback from 'react-native-bpk-component-touchable-native-feedback';
+import { colorBlue500, spacingBase } from 'bpk-tokens/tokens/base.react.native';
+
+const styles = StyleSheet.create({
+  pickerItem: {
+    padding: spacingBase,
+  },
+  selected: {
+    color: colorBlue500,
+  },
+});
+
+const BpkPickerItem = props => {
+  const { color, value, label, onPress, selected } = props;
+
+  const textStyle = selected ? styles.selected : {};
+  if (color && !selected) {
+    textStyle.color = color;
+  }
+
+  return (
+    <View style={styles.pickerItem}>
+      <BpkTouchableNativeFeedback
+        onPress={() => onPress && onPress(value)}
+        accessibilityLabel={label}
+      >
+        <BpkText style={textStyle}>{label}</BpkText>
+      </BpkTouchableNativeFeedback>
+    </View>
+  );
+};
+
+BpkPickerItem.propTypes = {
+  color: PropTypes.string,
+  label: PropTypes.string.isRequired,
+  value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  onPress: PropTypes.func,
+  selected: PropTypes.bool,
+};
+
+BpkPickerItem.defaultProps = {
+  color: null,
```

Comment:
`color` is it's own prop to match its iOS counterpart.

Reply:
I understand you have added this to the android picker item to align with the native behaviour in iOS. But lets drop it and remove it as an explicit prop. Folks can still implement this in userland in both Android and iOS by virtue of ...rest